lib/extensions/base.py

```diff
@@ -137,8 +137,7 @@ def descendants(self, node, selected=False, troubleshoot=False):  # noqa: C901
             if len(list(layer_commands(node, "ignore_layer"))):
                 return []
 
-        if (node.tag in [EMBROIDERABLE_TAGS, SVG_GROUP_TAG] and
-                element.get_style('display', 'inline') is None):
+        if (node.tag in EMBROIDERABLE_TAGS or node.tag == SVG_GROUP_TAG) and element.get_style('display', 'inline') is None:
```

Comment:
Are you enforcing line length limits at all in code base?

Noting I ran this specific change locally and my stitch plan looks sane. 

Only the same 7 color blocks and elements I expect